server/athenian/api/openapi/openapi.yaml

```diff
@@ -838,6 +838,219 @@ paths:
+                $ref: '#/components/schemas/GenericError'
+          description: The specified team was not found.
+      security:
+      - bearerAuth: []
+      summary: List the team's members. The user must belong to the account that owns the
+        team.
+      tags:
+      - team
+# {% if False %}
+      - default
+# {% endif %}
+      x-openapi-router-controller: athenian.api.controllers.team_controller
+      x-codegen-request-body-name: body
+    put:
+      operationId: update_team
+      parameters:
+      - description: Numeric identifier of the team to update.
+        explode: false
+        in: path
+        name: id
+        required: true
+        schema:
+          type: integer
+        style: simple
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Team'
+        x-body-name: body
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                type: object
+          description: Empty object indicates a successful operation.
+        403:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The user is forbidden to update the specified team.
+        404:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The specified team was not found.
+        409:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: There is an existing team with the same contents.
+      security:
+      - bearerAuth: []
+      summary: Update a team.
+      tags:
+      - team
+# {% if False %}
+      - default
+# {% endif %}
+      x-openapi-router-controller: athenian.api.controllers.team_controller
+      x-codegen-request-body-name: body
+  /teams/{id}:
+    get:
+      operationId: list_teams
+      parameters:
+      - description: Numeric identifier of the account.
+        explode: false
+        in: path
+        name: id
+        required: true
+        schema:
+          type: integer
+        style: simple
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TeamList'
+          description: List the teams belonging to the specified account.
+        404:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The user does not belong to the specified account.
+        409:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: Concurrent requests that are finishing the account registration.
```

Comment:
mhmmm tricky. tbh I added this to make as much similar to `/reposets/{id}`, but probably in this case even `200` with empty list would fit as well. I think that `409` in case of non-finished account registration fits for enpdoints involving write ops.

Reply:
Agree on returning empty list for read ops, and error on write ones.